media-source-respec.html

```diff
@@ -436,25 +438,13 @@
               <p class="note"><a def-id="appendBuffer"></a>, <a def-id="appendStream"></a> and <a def-id="endOfStream"></a> can update the duration under certain circumstances.</p>
             </li>
           </ol>
-        </dd>
-
-        <dt>attribute EventHandler onsourceopen</dt>
-        <dd>
+        </dd><dt><dfn><code>onsourceopen</code></dfn> of type <span class="idlAttrType"><a>EventHandler</a></span></dt><dd>
           <p>The event handler for the <a def-id="sourceopen"></a> event.</p>
-        </dd>
-
-        <dt>attribute EventHandler onsourceended</dt>
-        <dd>
+        </dd><dt><dfn><code>onsourceended</code></dfn> of type <span class="idlAttrType"><a>EventHandler</a></span></dt><dd>
           <p>The event handler for the <a def-id="sourceended"></a> event.</p>
-        </dd>
-
-        <dt>attribute EventHandler onsourceclose</dt>
-        <dd>
+        </dd><dt><dfn><code>onsourceclose</code></dfn> of type <span class="idlAttrType"><a>EventHandler</a></span></dt><dd>
           <p>The event handler for the <a def-id="sourceclose"></a> event.</p>
-        </dd>
-
-        <dt>SourceBuffer addSourceBuffer(DOMString type)</dt>
-        <dd>
+        </dd></dl></section><section><h2>Methods</h2><dl class="methods" data-dfn-for="MediaSource" data-link-for="MediaSource"><dt><dfn><code>addSourceBuffer</code></dfn></dt><dd>
```

Comment:
Ditto: DOMString (param type) and SourceBuffer (retval type) are missing links in index.html